.github/mergify.yml

```
@@ -2,0 +1,552 @@
+  is_a_bot: &is_a_bot
+    - or:
+      - "author=cloudpossebot"
+      - "author=github-actions[bot]"
+      - "author=dependabot[bot]"
+      - "author=renovate[bot]"
+      - "author=cloudposse-releaser[bot]"
+
+  # Not a bot user
+  not_a_bot: &not_a_bot
+    - and:
+      - "-author=cloudpossebot"
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)